models_config.truefoundry.yaml

```diff
@@ -1,10 +1,21 @@
 model_providers:
-  - provider_name: truefoundry
-    api_format: openai
-    base_url: https://llm-gateway.truefoundry.com/api/inference/openai
-    api_key_env_var: TFY_API_KEY
-    llm_model_ids:
-      - "openai-main/gpt-4-turbo"
-      - "openai-main/gpt-3-5-turbo"
-    embedding_model_ids:
-      - "openai-main/text-embedding-ada-002"
+    - provider_name: truefoundry
+      api_format: openai
+      base_url: https://llm-gateway.truefoundry.com/api/inference/openai
+      api_key_env_var: TFY_API_KEY
+      llm_model_ids:
+          - "openai-main/gpt-4-turbo"
+          - "openai-main/gpt-3-5-turbo"
+      embedding_model_ids:
+          - "openai-main/text-embedding-ada-002"
+      reranking_model_ids: []
+
+    - provider_name: local-infinity
+      api_format: openai
```

Comment:
the api format is not correct? Shouldn't this be infinity?

Reply:
openai is used for embeddings

Reply:
Then there should be two entries in models_config json file.